openlcs/libs/corgi_handler.py

```diff
@@ -57,35 +85,41 @@ def parse_component_link(self, component_link):
         component = {}
         unquote_component_link = self.unquote_link(component_link)
         rpm_pattern = re.compile(
-            r'.*=pkg:(?P<type>.*)/redhat/(?P<name>.*)@(?P<version>.*)-(?P<release>.*)\?arch=.*')  # noqa
+            r'.*=pkg:(?P<type>.*)/redhat/(?P<name>.*)@(?P<version>.*)-(?P<release>.*)\?arch=(?P<arch>.*)')  # noqa
         remote_source_pattern = re.compile(
             r'.*=pkg:(?P<type>.*?)/(?P<name>.*)@(?P<version>.*)')
         match = rpm_pattern.match(unquote_component_link)
         if match:
             component = {
+                'uuid': str(uuid.uuid4()),
```

Comment:
if the components obtained from this module has inconsistent uuid with corgi's, it has to be marked as "synced=False". I don't see this in "create_component", no?

Reply:
updated